test/images/agnhost/netexec/netexec.go

```diff
@@ -693,7 +693,11 @@ func startSCTPServer(sctpPort int) {
 	for {
 		conn, err := listener.AcceptSCTP()
 		assertNoError(err, fmt.Sprintf("failed accepting SCTP connections"))
-		clientAddress := conn.RemoteAddr().String()
+		remoteAddr, err := conn.SCTPRemoteAddr(0)
+		if err != nil {
+			assertNoError(err, "failed to get SCTP client remote address")
+		}
+		clientAddress := remoteAddr.String()
```

Comment:
you should check that remoteAddrs is not nil

Reply:
I think this should be sufficient `remoteAddrs` will only be nil if err is not nil.